### PR TITLE
Add archive/unarchive cards

### PR DESCRIPTION
**Pull request by dhaselhan**
* Add ability to archive / unarchive cards

### CODE REVIEW

File: alcs-frontend/src/app/features/board/dialogs/planning-review/planning-review-dialog.component.ts [hunk 1]
```diff
@@ -67,6 +70,13 @@ export class PlanningReviewDialogComponent implements OnInit, OnDestroy {
       }
     });
 
+    this.authService.$currentUser.pipe(takeUntil(this.$destroy)).subscribe((currentUser) => {
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/board/dialogs/planning-review/planning-review-dialog.component.ts:73]: I almost want to move this check to some kind CardDialogBaseComponent

File: services/apps/alcs/src/admin/unarchive-card/unarchive-card.service.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  ) {}
+
+  async fetchByFileId(fileId: string) {
+    const result: {
+      cardUuid: string;
+      type: string;
+      status: string;
+      createdAt: number;
+    }[] = [];
+    const application = await this.applicationService.getDeletedCard(fileId);
```
Comment (mhuseinov) [inline services/apps/alcs/src/admin/unarchive-card/unarchive-card.service.ts:25]: This looks like a good candidate to split into multiple functions. It should be easier later to keep track of all the different card types that get attached to the result list

**Review by mhuseinov — APPROVED** [review]

File: alcs-frontend/src/app/features/board/dialogs/planning-review/planning-review-dialog.component.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/board/dialogs/planning-review/planning-review-dialog.component.ts]: Done!

File: services/apps/alcs/src/admin/unarchive-card/unarchive-card.service.ts (hunk 2 above)
Comment (dhaselhan) [reply services/apps/alcs/src/admin/unarchive-card/unarchive-card.service.ts:25]: Done!

**Review by mhuseinov — APPROVED** [review]
🔥

**Pull request merged**